### ISSUE TITLE
Add the concept to stylesheets.

### ISSUE THREAD

**Issue by LukyVj** [bullgit/SCREAM.CSS#2]
Yo, 

I decided to open an issue, since I'm too lazy to clone, and send a pull request. 
BUT ! I had an idea, that could extend the concept. 

SCREAM.CSS is cool, obviously, but how to make it cooler ? Well, my solution is to use it directly into codes ! Like in css ! 

Regular CSS : 

```
.selector{ 
    property: value;
}
```

SCREAM CSS : 

```
.SELECTOR{
    PROPERTY: VALUE;
}
```

What is the reason ? There is none. You still could say : 

> YEAH I WAS PISSED OF SO I CODED MY WHOLE PROJECT WITH THE SCREAM CSS METHODOLOGY BRO

**Comment by TimPietrusky** [comment]
+1

**Comment by reimersjan** [comment]
@LukyVj I already did that in [MASTER/SCREAM.CSS](https://github.com/bullgit/SCREAM.CSS/blob/MASTER/SCREAM.CSS). Or what do you mean?

**Comment by LukyVj** [comment]
Yes you did, but my point was more to "explain" to the users, that it's also a methodology. 

It's not just a "client front" effect, if you are perfectionnist, along using SCREAM.CSS for the visual rendering, you can also write the code with the SCREAM METHODOLOGY. 

Which is : CAPS LOCK EVERYWHERE ! 

---

Anyone can do it be default, but I like the idea to call it a "methodology", you know, since BEM, SUIT, ect are mainstream, we could tweet something like : `THERE IS A NEW CSS METHODO, ITS THE SCREAM METHOD ! FORGET ABOUT BEM, SCREAM EVERYWHERE !`

**Comment by reimersjan** [comment]
Now I get it. NOICE!

+1

Should be a new paragraph in the README.MD

**Comment by LukyVj** [comment]
Yep, I say this in gitter, just a new readme entry !
